frontend/src/components/maps/MapFilterBar.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import * as MOCK from 'mocks/filterDataMock';
+import { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme from 'enzyme';
+import MapFilterBar from './MapFilterBar';
+import LookupCodeDropdown from 'components/common/LookupCodeDropdown';
+import renderer from 'react-test-renderer';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const handleAgencyChanged = jest.fn();
+const handlePropertyClassificationChanged = jest.fn();
+
+const filterBar = shallow(
+  <MapFilterBar
+    agencyLookupCodes={MOCK.AGENCIES}
+    propertyClassifications={MOCK.CLASSIFICATIONS}
+    onSelectAgency={handleAgencyChanged}
+    onSelectPropertyClassification={handlePropertyClassificationChanged}
+  />,
+);
+
+// Could change as project progresses
+it('Currently has 2 drop downs, one for agency one for classifications', () => {
```

Comment:
I'd be tempted to let the snapshot handle tests like this in the future, but feel free to leave this in.

Reply:
Thought about that afterwards, I agree. Will let snapshots handle issues like that from now on!